diff -uprN tss.orig/utils/makefile-common tss/utils/makefile-common
--- tss.orig/utils/makefile-common	2019-04-24 19:48:55.000000000 +0200
+++ tss/utils/makefile-common	2020-07-08 12:37:29.210793571 +0200
@@ -45,7 +45,8 @@ CCFLAGS += 				\
 	-Wall -W -Wmissing-declarations -Wmissing-prototypes -Wnested-externs \
 	-Wformat=2 -Wold-style-definition -Wno-self-assign \
 	-Werror=declaration-after-statement \
-	-ggdb -O0 -c
+	-ggdb -c -DTPM_INTERFACE_TYPE_DEFAULT="\"dev\"" \
+	-DTPM_DEVICE_DEFAULT="\"/dev/tpmrm0\""
 
 # to compile with optimizations on (warning will result)
 #	-O3 -c
@@ -56,6 +57,8 @@ CCFLAGS += 				\
 
 #LNFLAGS += 	-ggdb
 
+LNLLIBS += -lcrypto
+
 ALL += 	$(LIBTSS)				\
 	$(LIBTSSA)				\
 	$(LIBTSSUTILS)
